go/cmd/dolt/commands/diff.go

```diff
@@ -201,6 +211,14 @@ func parseDiffArgs(ctx context.Context, dEnv *env.DoltEnv, apr *argparser.ArgPar
 	switch strings.ToLower(f) {
 	case "tabular":
 		dArgs.diffOutput = TabularDiffOutput
+		switch strings.ToLower(apr.GetValueOrDefault(DiffMode, "line")) {
```

Comment:
Not sure about changing this default. Maybe let's keep it the same for now (row based) and give the team a little time to play with it.

Reply:
On consideration, I think I'm fine with this. It will only differ from row based in the case that one of the columns being diffed is a text column with a newline in it, correct?